### PR TITLE
Create deploy script and park_king python package

### PR DESCRIPTION
**Pull request by Dano-drevo**

### CODE REVIEW

File: park_king.egg-info/PKG-INFO [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        
+        Frontend:
+        --
+        1. Go to root directory of FE project (park-king-fe) and install dependencies `npm install`
+        2. Modify endpoint property in environments variables file (park-king-fe/src/enviroment.ts) with actual backend IP address <BACKEND_IP>
+        3. Run web version of app from root of FE project (park-king-fe) `ionic serve`
+        
+Keywords: app,mobile,flask
+Platform: UNKNOWN
+Classifier: License :: Proprietary License :: Mavenir License
```
Comment (herodotos) [inline park_king.egg-info/PKG-INFO]: Urcite by som tu nazov firmy neuvadzal.

File: park_king/park-king-be/notifications.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python3
+from pyfcm import FCMNotification
+from flask import jsonify
+
+import logging
+
+def send_notifications(registration_ids):
+    if registration_ids[0] is not None:
```
Comment (herodotos) [inline park_king/park-king-be/notifications.py]: Toto by si mozno mohol zmenit, podla toho co si mi navrhoval v sprave registration_ids vracia [None]

File: park_king.egg-info/PKG-INFO (hunk 1 above)
Comment (Dano-drevo) [reply park_king.egg-info/PKG-INFO]: jaj sry,, to som zabudol zmazat,  idem to commitnut a mergnem

File: park_king/park-king-be/notifications.py (hunk 2 above)
Comment (Dano-drevo) [reply park_king/park-king-be/notifications.py]: trochu som to zrefaktoroval, pushnem trz...skus mi to otestovat este ked tak ak mozes pls

Comment (herodotos) [reply park_king/park-king-be/notifications.py]: No tazko, najskor ked pridem domov.

Comment (Dano-drevo) [reply park_king/park-king-be/notifications.py]: dobre, tak to mergnem, funkcionalitu to len prida takze by to nemalo nic rozbit co este nebolo rozbite

**Comment by herodotos** [comment]
Manually tested and no new bugs found.

**Pull request merged**